src/components/c-modal.vue

```diff
@@ -236,7 +236,6 @@
     // deactivated() {},
     beforeDestroy() {
       this.$modalStack.remove(this.uuid);
-      enableBodyScroll(document.body);
```

Comment:
todo: We should keep this one. In cases where the component is used with a `v-if` it might be removed from the DOM before the `open` state changed. Removing this line would result in the body scroll still be missing.